internal/introspection/router.go

```diff
@@ -32,6 +37,16 @@ type Procedure struct {
 	RPCType   string `json:"rpcType"`
 }
 
+// ProcedureName outputs a encoding-native procedure name.
+func (p Procedure) ProcedureName() string {
+	// see transport/grpc/util.go#toFullMethod
+	if p.Encoding == proto {
```

Comment:
You should use the const declared in the encoding/protobuf pkg here.

```
if p.Encoding == protobug.Encoding {
}
```

Also, this should also be the case with encoding==json no?

Reply:
I did not import the const on purpose. If we do that, people who only want to import our debug code (which uses this introspection pkg), will get the whole protobuf pkg. I think it is too much of a side effect for a single const. Not sure about json, json here could be http/json or grpc/json. I would only consider proto for now.

Reply:
I have just checked for http/json (from proto) and you need to pass `::` (for instance `rpc-procedure: uber.infra.net.rpc.probe.Probe::Echo`), so this is good.